sdk/formrecognizer/Azure.AI.FormRecognizer/api/Azure.AI.FormRecognizer.netstandard2.0.cs

```diff
@@ -0,0 +1,378 @@
+        {
+            V2_0 = 1,
+        }
+    }
+    public static partial class OperationExtensions
+    {
+        public static System.Threading.Tasks.Task<Azure.Response<Azure.AI.FormRecognizer.Models.FormPageCollection>> WaitForCompletionAsync(this System.Threading.Tasks.Task<Azure.AI.FormRecognizer.Models.RecognizeContentOperation> operation, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public static System.Threading.Tasks.Task<Azure.Response<Azure.AI.FormRecognizer.Models.RecognizedFormCollection>> WaitForCompletionAsync(this System.Threading.Tasks.Task<Azure.AI.FormRecognizer.Models.RecognizeCustomFormsOperation> operation, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public static System.Threading.Tasks.Task<Azure.Response<Azure.AI.FormRecognizer.Models.RecognizedFormCollection>> WaitForCompletionAsync(this System.Threading.Tasks.Task<Azure.AI.FormRecognizer.Models.RecognizeReceiptsOperation> operation, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public static System.Threading.Tasks.Task<Azure.Response<Azure.AI.FormRecognizer.Training.CustomFormModelInfo>> WaitForCompletionAsync(this System.Threading.Tasks.Task<Azure.AI.FormRecognizer.Training.CopyModelOperation> operation, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public static System.Threading.Tasks.Task<Azure.Response<Azure.AI.FormRecognizer.Training.CustomFormModel>> WaitForCompletionAsync(this System.Threading.Tasks.Task<Azure.AI.FormRecognizer.Training.TrainingOperation> operation, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+    }
+    public partial class RecognizeContentOptions
+    {
+        public RecognizeContentOptions() { }
+        public Azure.AI.FormRecognizer.FormContentType? ContentType { get { throw null; } set { } }
+    }
+    public partial class RecognizeCustomFormsOptions
+    {
+        public RecognizeCustomFormsOptions() { }
+        public Azure.AI.FormRecognizer.FormContentType? ContentType { get { throw null; } set { } }
+        public bool IncludeFieldElements { get { throw null; } set { } }
+    }
+    public partial class RecognizeReceiptsOptions
+    {
+        public RecognizeReceiptsOptions() { }
+        public Azure.AI.FormRecognizer.FormContentType? ContentType { get { throw null; } set { } }
+        public bool IncludeFieldElements { get { throw null; } set { } }
+    }
+}
+namespace Azure.AI.FormRecognizer.Models
+{
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct BoundingBox
+    {
+        private readonly object _dummy;
```

Comment:
I just realized that we are using this term in our library.
Could we replace it with `placeholder` ? 
Same applies for other types

Reply:
This file is autogenerated by https://github.com/dotnet/arcade/tree/master/src/Microsoft.DotNet.GenAPI tool. We can file an issue on that repo if we think it's worth changing.

Reply:
I am ok with filling an issue and doing it in a different PR. I thought maybe we  could also just add it here as it doesn't seem that big.
I do believe we need to change this before we GA

Reply:
> I am ok with filling an issue and doing it in a different PR. I thought maybe we could also just add it here as it doesn't seem that big.

The file is completely autogenerated including this private field. We do the same for all other libraries and so does dotnet core as well.

Reply:
I should have been more clear, I meant making the change to those types in this open PR instead of creating a new one.

Reply:
Discussed offline: this won't block GA since this is autogenerated, and it's not a term used by us in our library. We can file an issue in the [dotnet/arcade](https://github.com/dotnet/arcade/tree/master/src/Microsoft.DotNet.GenAPI) repo, as suggested, or try to handle it on our side, if feasible.